tests/subsys/net/lib/fota_download/prj.conf

```
@@ -8,3 +8,4 @@ CONFIG_ZTEST_NEW_API=y
 CONFIG_FLASH=y
 CONFIG_FW_INFO=y
 CONFIG_TFM_BUILD_NS=y # In order to mock tfm_platform_s0_active
+CONFIG_SYS_HASH_FUNC32_CHOICE_MURMUR3=y
```

I think it is enough to add

```suggestion
CONFIG_SYS_HASH_FUNC32=y
```

---

Fixed